### DISCUSSION TITLE
Add custom domain to localhosted supabase

### DISCUSSION THREAD

**Discussion by Prains** [coollabsio/coolify#2535] [Q&A]
Currently hosting my frontend and supabase with coolify. frontend is a docker container using port 3000. when forwarding my main domain to server ip and setting domain in environment settings, everything works good. frontend is accesible and ssl is working.

when hosting supabase, i don't know how to put it on subdomain to be accesible with domain and to use SSL from coolify. i changed domain in domains settings in supabase environment but i don't know on which ip or which port make a record in DNS server of my domain.

can somebody help me and explain how to assign a custom domain to supabase that is being hosted on localhost?

**Comment by roo12312** [comment]
even i want to know

**Comment by Prains** [comment]
solved this very simply:

any subdomain and domains that coolify should use need to be recorded in DNS with coolify server. coolify proxy will handle rest
